account_voucher_supplier_invoice_number/voucher.py

```diff
@@ -40,10 +40,10 @@ def _get_supplier_invoice_number(self, cr, uid, ids, name, args, context=None):
     
     _columns = {
         'supplier_invoice_number': fields.function(_get_supplier_invoice_number,
-            type='char', size=64, string="Supplier Invoice Number"),
+            type='char', size=64, string=_("Supplier Invoice Number")),
         }
 
-class voucher(orm.Model):
+class voucher(models.Model):
     _inherit = 'account.voucher'
     
     def recompute_voucher_lines(self, cr, uid, ids, partner_id, journal_id, price,
```

Comment:
Ya, I know, but when I change it all like there (only  _get_supplier_invoice_number):
https://github.com/JehoG/account-payment/blob/Example/account_voucher_supplier_invoice_number/voucher.py
it's just leaving the field blank (in the form view fo a supplier payment under 'supplier invoice number'. But if I check at the value of res just before the line 40, I got what it's suppose to be.
So I got told to push it only with those minor changes if it's working.
You see something that I missed in this example?